src/main/java/io/orkes/conductor/client/ApiClient.java

```diff
@@ -308,22 +308,22 @@ public ApiClient setKeyManagers(KeyManager[] managers) {
     }
 
     public ApiClient setDateFormat(DateFormat dateFormat) {
-        this.json.setDateFormat(dateFormat);
+        LOGGER.warn("This feature is deprecated");
         return this;
     }
 
     public ApiClient setSqlDateFormat(DateFormat dateFormat) {
-        this.json.setSqlDateFormat(dateFormat);
+        LOGGER.warn("This feature is deprecated");
```

Comment:
We should also point them to ObjectMapper instance which they need to modify, and mark the methods as `@Deprecated` so that they are highlighted by the IDEs